docs/source/developer_guide/core/internals.rst

```diff
@@ -61,11 +61,13 @@ The :py:class:`~aiida.orm.nodes.Node` can be annotated with labels, description
 
 *Label management:*
 
-- :py:meth:`~aiida.orm.nodes.Node.label` returns the label of the node and can be used as a setter property.
+- :py:attr:`~aiida.orm.nodes.Node.label` returns the label of the node.
+  It can also be used to *change* the label, e.g. ``mynode.label = "new label"``.
 
 *Description management:*
 
-- :py:meth:`~aiida.orm.nodes.Node.description`: the description of the node (more detailed than the label) and can be used as a setter property.
+- :py:attr:`~aiida.orm.nodes.Node.description`: returns the description of the node (more detailed than the label).
+  It can also be used to *change* the description, e.g. ``mydnode.description = "new description"``.
```

Comment:
If you decide to do some more changes there is a typo: ``mydnode`` should be ``mynode`` I guess

Reply:
I believe this is outdated (`mydnode` no longer in source + document removed)

Reply:
I think you have misunderstood. You added this line, and should simply remove the 'd':

```suggestion
  It can also be used to *change* the description, e.g. ``mynode.description = "new description"``.
```

Reply:
thanks - very strange, I checked out the branch locally and did not find the string. only when I pulled from the PR on my fork.
Anyhow, it's fixed now.